### PR TITLE
[Fil-480] Add validation to add storage providers input

### PR DESCRIPTION
**Pull request by Filip-L**

### CODE REVIEW

File: src/components/cards/dialogs/allowedSps/AllowedSps.tsx
Comment (kacperzuk-neti) [inline src/components/cards/dialogs/allowedSps/AllowedSps.tsx:101]: Cleanup?

File: src/components/cards/dialogs/allowedSps/AllowedSps.tsx
Comment (kacperzuk-neti) [inline src/components/cards/dialogs/allowedSps/AllowedSps.tsx:108]: @tplocic20 what do you think about DYI form validation and debouncing? Is this the best way given the state of the app or is there some good lightweight lib we could include?

**Review by kacperzuk-neti — APPROVED** [review]
Get approval from @tplocic20 before merging please.

File: src/components/cards/dialogs/allowedSps/AllowedSps.tsx
Comment (tplocic20) [reply src/components/cards/dialogs/allowedSps/AllowedSps.tsx:107]: Ideally we would use something like hookForm - but it's too much of a refactor I'm afraid

**Review by tplocic20 — APPROVED** [review]

**Pull request merged**
